README.md

```diff
@@ -43,13 +39,13 @@ Format code according to the [Rust style Guide](https://github.com/rust-dev-tool
 
 GH Actions are located here: [.github/workflows](.github/workflows)
 
-- cargo docs  
+- cargo docs
```

Comment:
@BearGuy we need these two spaces `  `. without those the list line-break will not work

Reply:
current state:  
![image](https://user-images.githubusercontent.com/1016218/118218417-f0f51b80-b4a9-11eb-87e5-9431ec5a7cb6.png)

Reply:
do you use a markdown-formatter in your ide?